### DISCUSSION TITLE
Issue in sending content for Review & Publish

### DISCUSSION THREAD

**Discussion by vayu068** [Sunbird-Ed/Community#230] [Installation]
Hello community,
I have a working portal with all forms created as mentioned in other threads. 
I am able to login into the console & access the Workspace . Please find below sequence of screenshots to understand the issue:
1. Upload content: PDF file uploaded & click on save button.
![Screenshot 2022-10-30 at 12 53 22 AM](https://user-images.githubusercontent.com/8116886/198849610-399e2604-db94-4c6e-8c64-2e6151d5d472.png)
2. I understand that after saving it should be available in Drafts section which is not the case here.
![Screenshot 2022-10-30 at 12 57 16 AM](https://user-images.githubusercontent.com/8116886/198849662-8062ea46-d458-4103-8b28-9a24c9ad5594.png)
3.  Clicked on Course & got this error
![Screenshot 2022-10-30 at 12 50 03 AM](https://user-images.githubusercontent.com/8116886/198849731-bc382c3f-1c18-40c0-9b32-05e1770fa800.png)
4. Created a resource & tried sending it for Review. Got following error. (It seems license type in web page & backend are not matching)
![Screenshot 2022-10-30 at 12 20 51 AM](https://user-images.githubusercontent.com/8116886/198849787-65b41695-c69e-47cf-8207-040182be826a.png)
5. Tried putting Youtube URL and got this error. (Seems somewhere i need to put youtube token)
![Screenshot 2022-10-30 at 12 23 21 AM](https://user-images.githubusercontent.com/8116886/198849801-8243107d-e1c9-4175-82e0-8ea993447a00.png)
6. Using Postman, submitted the uploaded content for Review via content/v1/review/ API &  read content from via content/v1/read API. (Got successful results with status set to Review. Still on UI no content is available for under Review section)
<img width="1120" alt="Screenshot 2022-10-30 at 12 47 14 AM" src="https://user-images.githubusercontent.com/8116886/198849866-8223eee3-6c16-4c19-816f-39d2273c1ecf.png">
7. Tried publishing the content via content/v1/publish API, getting Token Invalid error although using same token, earlier APIs worked.

<img width="890" alt="Screenshot 2022-10-30 at 12 47 39 AM" src="https://user-images.githubusercontent.com/8116886/198850020-b2a0652d-f8c7-431f-99cc-6536a580f84d.png">

Please let me know what logs are required to further diagnose the issue.

**Comment by beepdot** [comment]
@vayu068 I would request you to please create individual discussion topics for each as it would be easier to answer and also benefit the community. Once I provide you some answers here, each item will start moving in a different direction and it would become difficult to track.
When you create new discussions, please tag me @keshavprasadms 
Point # 1 - Please ignore the error, that is a know error and it will cause no issues.
You can combine point # 6 and # 7 into one discussion.

**Reply by vayu068** [reply]
Sure, I will create separate threads. 
Created separate thread https://github.com/orgs/Sunbird-Ed/discussions/236 for point 6 & 7

**Reply by vayu068** [reply]
@keshavprasadms , any updates on other issues apart from point 6 & 7?
Please let me know if any further logs/inputs required from my side.

**Reply by beepdot** [reply]
@vayu068 We will respond in the respective threads.
@sharathkashyap Please close this thread.
